source/scripts/models/Hero.js

```diff
@@ -206,6 +212,8 @@ export default class Hero extends Pixi.Sprite {
             }
         }
 
+        this.collider.update(this.position, {x: this.scale.x, y: this.scale.y}, this.rotation)
```

Comment:
Would it eventually make sense for the collider to use `this.parent.position` as a reference, instead of manually updating the collider with their own copy of the collider.